alacritty.yml

```diff
@@ -450,6 +468,27 @@
     # binding section.
     #modifiers: None
 
+# Regex hints
+#
+# Terminal hints can be used to find text in the visible part of the terminal
+# and pipe it to other applications.
+#hints:
+  # Keys used for the hint labels
+  #alphabet: "jfkdls;ahgurieowpq"
+
+  # List with all available hints
+  #
+  # The fields `command`, `binding.key` and `binding.mods` accept the same
+  # values as they do in the `key_bindings` section.
+  #
+  # Example:
+  #  - regex: "alacritty/alacritty#\\d*"
```

Comment:
I think mentioning the `enabled` in the docs is a good idea, though I don't like showing it like it's a default rather than an example, since there is no default.

Though generally this might be made much easier when we transition to use this for URLs with a default already present. In that case we can do it the way you did it here.